system/jhtree/jhinplace.cpp

```diff
@@ -53,6 +53,9 @@ static constexpr byte NSFcompressTrailing   = 0x20;     // compressed payload ha
 static constexpr byte NSFscaleN             = 0x40;     // filepositions are scaled by size N - useful for filepositions in fixed size files
 static constexpr byte NSFscaleFilepos       = 0x80;     // filepositions are scaled by nodesize
 
+static constexpr byte OFsequential          = 0x40;     // Options are a sequential range of values e.g. '0','1','2','3','4','5'.  Only first value stored
```

Comment:
Should clarify that only first non-space value is stored?

Reply:
clarified